ui/src/api/api.ts

```diff
@@ -12,6 +13,12 @@ import {
   PaperIdWithEntityCounts,
 } from "./types";
 
+const token = cookie.parse(document.cookie)["readerAdminToken"];
```

Comment:
How does one set this adminToken locally? Would they open the cookie editor in the browser dev tools? I wonder if it we possible to let us set the reader admin token through environment variables?